lib/components/CardCarousel/CardCarousel.tsx

```diff
@@ -0,0 +1,127 @@
+import { Carousel as ArkCarousel, useCarousel } from '@ark-ui/react/carousel';
+import React from 'react';
+import { MdChevronLeft, MdChevronRight } from 'react-icons/md';
+
+export interface CardCarouselProps {
+  items?: React.ReactNode[];
+  translations: {
+    goToPage: string;
+    previousPage: string;
+    nextPage: string;
+  };
+}
+export const CardCarousel = ({ items = [], translations }: CardCarouselProps) => {
+  // Using the useCarousel hook in order to use the RootProvider of the carousel,
+  // which is required to allow calling the scrollTo methods on the carousel.
+  const carousel = useCarousel();
+  const rootRef = React.createRef<HTMLDivElement>();
+  const CARD_WIDTH = 260; // This needs to match the width of the MediaCard component
```

Comment:
I don't like the fact that the carousel needs to know the card width, but I could not figure out a good solution to calculate it on the fly. Is this ok or should it be a prop? Maybe the gap (between cards) should be a prop also?

Reply:
Not sure if carousel would be used also in other content also. 
So maybe `width` could be a prop. 

The gap should be same not matter what the content would be, in my opinion.

Reply:
I turned the width into a prop. Gap left as is.